etl/lib/multichannel_mip.py

```diff
@@ -0,0 +1,211 @@
+"""
```

Comment:
May be better merged with mip.py. Works better if mip.py is split into transforms / GCS handling / etc. And then multichannel_mip.py uses those same files.